src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import { Main, GridItems } from "../../components";
+
+class HomePage extends Component {
```

Comment:
Make it functional component
```suggestion
const HomePage = ()=> {
```